numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -29,9 +30,13 @@ def _get_libdevice_path_decision():
         ('Conda environment', get_conda_ctk()),
         ('Conda environment (NVIDIA package)', get_nvidia_libdevice_ctk()),
         ('CUDA_HOME', get_cuda_home('nvvm', 'libdevice')),
-        ('System', get_system_ctk('nvvm', 'libdevice')),
         ('Debian package', get_debian_pkg_libdevice()),
+        ('NVIDIA NVCC Wheel', get_libdevice_wheel()),
     ]
+    libdevice_ctk_dir = get_system_ctk('nvvm', 'libdevice')
```

Comment:
Why did we move the system toolkit after the Debian-packaged version? I think we want to preserve the order if we can.